spi-tester/src/main/java/org/ehcache/spi/test/SPITester.java

```diff
@@ -28,6 +29,25 @@ public Result runTestSuite() {
     Result result = new Result();
     result.testRunStarted();
     Class<? extends SPITester> testClass = getClass();
+    ArrayList<Method> beforeMethodList = new ArrayList<Method>();
+    ArrayList<Method> afterMethodList = new ArrayList<Method>();
+    for(Method m : testClass.getDeclaredMethods()){
+      if (m.isAnnotationPresent(Before.class)) {
+        beforeMethodList.add(m);
+      }
+      if (m.isAnnotationPresent(After.class)) {
+        afterMethodList.add(m);
+      }
+    }
+    if(!beforeMethodList.isEmpty()) {
```

Comment:
To match JUnit behavior and be consistent with introduced `@After`, the `@Before` methods should be executed for each test method.